arch/xtensa/include/xtensa_asm2_s.h

```diff
@@ -604,6 +604,11 @@ _Level\LVL\()Vector:
 	s32i a2, a1, ___xtensa_irq_bsa_t_a2_OFFSET
 	s32i a3, a1, ___xtensa_irq_bsa_t_a3_OFFSET
 
+#if defined(CONFIG_SOC_FAMILY_INTEL_ADSP) && defined(CONFIG_ADSP_IDLE_CLOCK_GATING)
```

Comment:
ADSP_IDLE_CLOCK_GATING depends on SOC_FAMILY_INTEL_ADSP, so there is no need for check on SOC_FAMILY_INTEL_ADSP

Reply:
Ack @nashif , updated in V2